regexes/client/browsers.yml

```diff
@@ -5,6 +5,13 @@
 # @license http://www.gnu.org/licenses/lgpl.html LGPL v3 or later
 ###############
 
+# CGNBrowser (https://www.cgn.it/)
```

Comment:
it doesn't look like a browser for constant use
![image](https://user-images.githubusercontent.com/1337066/108430781-f64c1c80-7252-11eb-8f65-e59d7e495be6.png)
we can add this as an `mobile app`,  name:  `Platform CGN`